ModelCatalogueGenomicsPlugin/grails-app/controllers/org/modelcatalogue/core/genomics/GenomicsController.groovy

```diff
@@ -1,24 +1,50 @@
 package org.modelcatalogue.core.genomics
 
-import org.modelcatalogue.core.DataClass
-import org.springframework.http.HttpStatus
-
+import org.modelcatalogue.core.DataClassService
+import org.modelcatalogue.core.DataModel
+import org.modelcatalogue.core.export.inventory.DataModelToDocxExporter
 /**
  * Controller for GEL specific reports.
  */
 class GenomicsController {
 
+    def assetService
+    DataClassService dataClassService
+
+    def imagePath = "http://www.genomicsengland.co.uk/wp-content/uploads/2015/11/Genomics-England-logo-2015.png"
+
+    def customTemplate = {
+        'document' font: [family: 'Calibri', size: 11], margin: [left: 20, right: 10]
+        'paragraph.title' font: [color: '#1F497D', size: 32.pt, bold:true], margin: [top: 150.pt, bottom: 10.pt]
+        'paragraph.subtitle' font: [color: '#1F497D', size: 36.pt], margin: [top: 0.pt]
+        'paragraph.description' font: [color: '#13D4CA', size: 16.pt, italic: true], margin: [left: 30, right: 30]
+        'heading1' font: [size: 18, bold: true]
+        'heading2' font: [size: 18, bold: true]
+        'heading3' font: [size: 16, bold: true]
+        'heading4' font: [size: 16, bold: true]
+        'heading5' font: [size: 15]
+        'heading6' font: [size: 14]
+        'table.row.cell.headerCell' font: [color: '#FFFFFF', size: 12.pt, bold: true], background: '#1F497D'
+        'table.row.cell' font: [size: 10.pt]
+        'paragraph.headerImage' height: 1.366.inches, width: 2.646.inches
+    }
+
+    def exportGelSpecification() {
 
-    def exportRareDiseases() {
-        DataClass dataClass = DataClass.get(params.id)
+        DataModel model = DataModel.get(params.id)
 
-        if (!dataClass) {
```

Comment:
it's a good habit to check for the model does not exist and return 404 status. please, add similar check for back.